test/Feature/AndOrParamsSpec.hs

```diff
@@ -33,13 +33,24 @@ spec actualPgVersion =
               {"id": 1, "child_entities": [ { "id": 1 }, { "id": 2 } ] }, { "id": 2, "child_entities": []},
               {"id": 3, "child_entities": []}, {"id": 4, "child_entities": []}
             ]|] { matchHeaders = [matchContentTypeJson] }
+        
         it "can do logic on the third level" $
-          get "/entities?child_entities.grandchild_entities.or=(id.eq.1,id.eq.2)&select=id,child_entities(id,grandchild_entities(id))" `shouldRespondWith`
-            [json|[
-              {"id": 1, "child_entities": [ { "id": 1, "grandchild_entities": [ { "id": 1 }, { "id": 2 } ]}, { "id": 2, "grandchild_entities": []}]},
-              {"id": 2, "child_entities": [ { "id": 3, "grandchild_entities": []} ]},
-              {"id": 3, "child_entities": []}, {"id": 4, "child_entities": []}
-            ]|] { matchHeaders = [matchContentTypeJson] }
+          get "/entities?child_entities.grandchild_entities.or=(id.eq.1,id.eq.2)&select=id,child_entities(id,grandchild_entities(id))"
+            `shouldRespondWith`
+              [json|[
+                {"id": 1, "child_entities": [
+                  { "id": 1, "grandchild_entities": [ { "id": 1 }, { "id": 2 } ]},
+                  { "id": 2, "grandchild_entities": []},
+                  { "id": 4, "grandchild_entities": []},
```

Comment:
In some cases I had to add rows to `data.sql` to be available for other tests, so existing tests that depend on what is in those tables had to be adjusted.